book/whats_new/20191124-swift-5.1-migration.md

```diff
@@ -0,0 +1,81 @@
+# Swift 5.1 Migration
+
+This article will go through recent changes in Zapp iOS SDK 13.0 and up made for the swift 5.1 migration. For the full SDK 13.0 changelog go [here](https://github.com/applicaster/Zapp-iOS/blob/development/CHANGELOG.md).
+
+
+## Internal Applicaster Plugin Migration
+
+Internal plugins should follow these instructions in order to update the creation of a closed framework.
+
+1. Install xcode 11.1 and cocoapods version 1.7.5 using the following command:
+    ```
+    sudo gem install cocoapods -v "=1.7.5"
+    ```
+
+2. Upgrade the following gems inside the Gemfile:
+    ```
+    gem 'xcpretty'
+    gem 'cocoapods', '= 1.7.5'
+    gem 'fastlane'
+    ```
+
+3. Update the Gemfile.lock by running `bundle install` in your main plugin directory.
+
+4. Go to circle-ci confog.yml file and change the following:
+    * Change the xcode version to 11.2.1
+    * Change the deploy version to 0.10.0
+    * Increment the caching keys to a new version, for example: 
```

Comment:
if there is a step:
`name: Ruby - set version`
 `command:  echo "ruby-2.4" > ~/.ruby-version`
need to update to 2.5

Reply:
done